extensions/resteasy-reactive/rest-client-reactive-jackson/runtime/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-rest-client-reactive-jackson-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
+    <name>Quarkus - MicroProfile Rest Client with Reactive RestEasy and Jackson</name>
+    <description>Jackson integration for RESTEasy Reactive MicroProfile Rest Client.</description>
```

Comment:
Very minor but we don't have dots for the others (and this appears at code.quarkus.io)

Reply:
I guess the `quarkus-rest-client-jackson` equivalent is `Enable Jackson serialization for the REST Client`, so we should harmonise those two

Reply:
the description is based on https://github.com/quarkusio/quarkus/blob/a1f4655416bac5b59a157b5840ce52c8c9d48015/extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/runtime/pom.xml#L14 which also has a comma at the end :)

I can change both descriptions to follow RestEasy classic. WDYT?

Reply:
Yes, they have to be harmonised, ideally.

Reply:
done